app/javascript/context/subreducers/SupplementalApplicationSubreducer.js

```diff
@@ -2,6 +2,10 @@ import LEASE_STATES from 'components/supplemental_application/utils/leaseSection
 
 import ACTIONS from '../actions'
 
+// This id is never sent to the backend, only used to identify frontend
+// "new" assistance objects that haven't been persisted yet.
+export const NEW_ASSISTANCE_PSEUDO_ID = -101
```

Comment:
No reason. Actually this should probably be a string since the others are strings, so this can be more descriptive like "new_assistance_pseudo_id"

I'll go ahead and make that change